packages/patternfly-4/react-table/src/components/Table/__snapshots__/Table.test.js.snap

```diff
@@ -3051,7 +3051,6 @@ exports[`Actions table 1`] = `
                                   </Item>,
                                 ]
                               }
-                              isGrouped={false}
```

Comment:
Wondering why this snapshot changed.

Reply:
I'm going to look into it. There are also approximately a million linting issues now in files I haven't touched in this PR, so I suspect it might be something from a different PR.